src/otx/algo/detection/backbones/fpn.py

```diff
@@ -0,0 +1,224 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Don't we need OTX license due to typing changes, including other files?

Reply:
If we just borrow whole file than we shouldn't add intel license
If we borrow implementation and modify it, we should add intel license and specify original source location

Reply:
Done!
2e709ad